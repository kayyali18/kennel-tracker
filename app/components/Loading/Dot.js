import styled from 'styled-components'

import BounceAnimation from './animation'

const Dot = styled.div`
  background-color: black;
  border-radius: 50%;
  width: 20px;
  height: 20px;
  margin: 0 15px;
  /* Animation */
  animation: ${BounceAnimation} 1s ease-in infinite;
  animation-delay: ${props => props.delay};
`

export default Dot
