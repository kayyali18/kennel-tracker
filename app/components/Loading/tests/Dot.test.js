import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

import Dot from '../Dot'

describe('<Dot />', () => {
  it('should match snapshot', () => {
    const renderedComponent = renderer.create(<Dot />).toJSON()
    expect(renderedComponent).toMatchSnapshot()
  })

  it('should render a div', () => {
    const renderedComponent = renderer.create(<Dot />).toJSON()
    expect(renderedComponent.type).toEqual('div')
  })
})
