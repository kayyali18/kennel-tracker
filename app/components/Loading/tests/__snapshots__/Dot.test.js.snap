// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Dot /> should match snapshot 1`] = `
.c0 {
  background-color: black;
  border-radius: 50%;
  width: 20px;
  height: 20px;
  margin: 0 15px;
  -webkit-animation: bJUCIX 1s ease-in infinite;
  animation: bJUCIX 1s ease-in infinite;
}

<div
  className="c0"
/>
`;
